### ISSUE TITLE
[System.ArgumentException] "No C#-Symbol.\r\nParametername: typeArguments"

### ISSUE THREAD

**Issue by TobiasSekan** [dotnet/roslyn#36857]
**Version Used**: 
Microsoft Visual Studio Professional 2019 Preview
Version 16.2.0 Preview 3.0
German Language

**Steps to Reproduce**:
1.  Open large code file
2.  Try to write a lambda inside a `Linq` Expression inside a object initializer

```csharp
IEnumerable<MyListObject> mylist;

// many code lines

var test = new TestObject
{
    MyProperty = mylist.Max(found => found  // <-- crash here
}
```

**Expected Behavior**:
Don't crash, and show completion

**Actual Behavior**:
 Crash on completion on the lambda property or after the lambda arrow
and restart Visual Studio

**Message**:
"Kein C#-Symbol.\r\nParametername: typeArguments"

**Stack Trace (with VS attach process)**
```
   bei Microsoft.CodeAnalysis.CSharp.Symbols.SymbolExtensions.EnsureCSharpSymbolOrNull[TSource,TDestination](TSource symbol, String paramName) in /_/src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs: Zeile336
   bei Microsoft.CodeAnalysis.CSharp.Symbols.MethodSymbol.Microsoft.CodeAnalysis.IMethodSymbol.Construct(ITypeSymbol[] arguments) in /_/src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs: Zeile1217
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.Instantiate(IMethodSymbol method, IList`1 invocationTypes) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile623
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.<>c__DisplayClass21_0.<InferTypeInArgument>b__2(IMethodSymbol m) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile564
   bei System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   bei System.Collections.Generic.List`1..ctor(IEnumerable`1 collection) in f:\dd\ndp\clr\src\BCL\system\collections\generic\list.cs: Zeile99
   bei System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypeInArgument(Int32 index, IEnumerable`1 methods, ArgumentSyntax argumentOpt) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile578
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypeInInvocationExpression(InvocationExpressionSyntax invocation, Int32 index, ArgumentSyntax argumentOpt) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile464
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypeInArgument(ArgumentSyntax argument, Nullable`1 previousToken) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile283
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypesWorker_DoNotCallDirectly(SyntaxNode node) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile184
   bei Microsoft.CodeAnalysis.LanguageServices.TypeInferenceService.AbstractTypeInferenceService.AbstractTypeInferrer.InferTypes(SyntaxNode expression, Boolean filterUnusable) in /_/src/Workspaces/Core/Portable/LanguageServices/TypeInferenceService/AbstractTypeInferenceService.AbstractTypeInferrer.cs: Zeile51
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypeInAnonymousFunctionExpression(AnonymousFunctionExpressionSyntax anonymousFunction) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile1555
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypeInLambdaExpression(LambdaExpressionSyntax lambdaExpression, Nullable`1 previousToken) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile1548
   bei Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypesWorker_DoNotCallDirectly(Int32 position) in /_/src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs: Zeile244
   bei Microsoft.CodeAnalysis.LanguageServices.TypeInferenceService.AbstractTypeInferenceService.InferTypes(SemanticModel semanticModel, Int32 position, String nameOpt, CancellationToken cancellationToken) in /_/src/Workspaces/Core/Portable/LanguageServices/TypeInferenceService/AbstractTypeInferenceService.cs: Zeile81
   bei Microsoft.CodeAnalysis.Shared.Extensions.ITypeInferenceServiceExtensions.InferType(ITypeInferenceService typeInferenceService, SemanticModel semanticModel, Int32 position, Boolean objectAsDefault, CancellationToken cancellationToken) in /_/src/Workspaces/Core/Portable/Shared/Extensions/ITypeInferenceServiceExtensions.cs: Zeile88
   bei Microsoft.CodeAnalysis.CSharp.Completion.Providers.EnumAndCompletionListTagCompletionProvider.<ProvideCompletionsAsync>d__1.MoveNext() in /_/src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs: Zeile76
```

**Comment by jcouv** [comment]
@jasonmalinowski fixed this recently. I'll let him link and close.

**Comment by jasonmalinowski** [comment]
Yep, fixed by #36517. Thanks @jcouv for the ping, and thanks @TobiasSekan for the bug -- if we didn't already have this fixed we would have been jumping on it right away.
